src/EventStore.Core/Services/Replication/LeaderReplicationService.cs

```diff
@@ -395,6 +395,13 @@ public class LeaderReplicationService : IMonitoredQueue,
 					if (replicationStart)
 						sub.SendMessage(new ReplicationMessage.ReplicaSubscribed(_instanceId, sub.SubscriptionId,
 							sub.LogPosition));
+
+					if (logPosition == chunk.ChunkHeader.ChunkStartPosition)
+						sub.SendMessage(new ReplicationMessage.CreateChunk(_instanceId,
+							sub.SubscriptionId,
+							chunk.ChunkHeader,
+							chunk.FileSize,
+							isCompletedChunk: false));
```

Comment:
To verify: is this always `false` ? as in some cases, this block may be entered when the chunk is scavenged

Reply:
fixed in: https://github.com/EventStore/EventStore/pull/4265

this PR needs to be rebased once the above PR is merged.